### ISSUE TITLE
DTS CE and SE should use similar UX to ChromeOS Firmware Utility

### ISSUE THREAD

**Issue by pietrushnic** [Dasharo/dasharo-issues#225]
**The problem you're addressing (if any)**

Current interface is confusing for end users. Explanation of the process which happen on the screen is not user friendly for most members of our community.

**Describe the solution you'd like**

We should leverage MrChromebox experience implemented in [scripts](https://github.com/MrChromebox/scripts). Please check slides here: https://mrchromebox.tech/#fwscript

We should maybe even hide more advanced options in submenu. We can probably do even better then in mentioned scripts but we should start with something reasonably scalable.

I would expect from person who will implement this feature to really compare UX between DTS CE and MrChromebox Firmware Utility Script.

**Where is the value to a user, and who might that user be?**

Simpler and more friendly looking interface. Extensible interface for future improvements.

**Describe alternatives you've considered**

Maybe simple ncurses interface would be better, since sooner or later we will transform into real application. There are also cool things written in Go and working as very nice terminal user interface applications. Of course we cannot go to far since application have to be usable over serial.

**Additional context**

**Comment by macpijan** [comment]
Some experiment was done here: https://github.com/Dasharo/dasharo-issues/discussions/361#discussioncomment-5177841

**Comment by macpijan** [comment]
Steps to achieve this:

- [ ] Move DTS into a separate repo first: https://github.com/Dasharo/dasharo-issues/issues/773
- [ ] Replace code for UI with the one from: https://github.com/Dasharo/dts/blob/v2/src/functions.sh

**Comment by DaniilKl** [comment]
> Replace code for UI with the one from:

@macpijan, the code was merged with DTS here: https://github.com/Dasharo/dts-scripts/pull/2. Does it cover your request? If so, can this issue be closed, or we are planning to continue developing DTS UI?

**Comment by macpijan** [comment]
Original request was from @pietrushnic but yest - I believe this is solved already.

**Comment by pietrushnic** [comment]
@macpijan @DaniilKl Yes, I couldn't be more happy to see it implemented after more than 2 years. Thank you.
